src/ripple/app/paths/impl/PaySteps.cpp

```diff
@@ -423,6 +429,12 @@ toStrandV2 (
         if (hasCurrency && hasIssuer &&
             isXRP(pe.getCurrency()) != isXRP(pe.getIssuerID()))
             return {temBAD_PATH, Strand{}};
+
+        if (hasIssuer && (pe.getIssuerID() == noAccount()))
```

Comment:
I see. So in the case where _just_ an Issuer is specified, why isn't its consistency checked?